src/Components/AddTodoForm.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import InputWithLabel from './InputWithLabel';
+import '../Styles/AddTodoForm.css';
+import PropTypes from "prop-types";
+
+const AddTodoForm = ({ onAddTodo }) => {
+
+    const [todoTitle, setTodoTitle] = React.useState("");
+
+
+    const handleTitleChange = (event) => {
+        let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    };
+
+
+    // handleAddTodo function makes a POST request to the api 
+    const handleAddTodo = (event) => {
+        // let _data =
```

Comment:
Go ahead and remove the commented code.  It will still be in source control if you need it.